fl4health/parameter_exchange/parameter_packer.py

```diff
@@ -71,3 +73,47 @@ def unpack_parameters(self, packed_parameters: NDArrays) -> Tuple[NDArrays, List
         model_parameters = packed_parameters[:split_size]
         param_names = packed_parameters[split_size:][0].tolist()
         return model_parameters, param_names
+
+
+class SparseCooParameterPacker(ParameterPacker[Tuple[NDArrays, NDArrays, List[str]]]):
+    """
+    This parameter packer is responsible for selecting an arbitrary set of parameters
+    and then representing them in the sparse COO tensor format, which requires knowing
```

Comment:
If you have a good documentation link for COO sparse representations, maybe you can also include a link 🙂